### ISSUE TITLE
NavigationBar - I can't remove the shadow/border

### ISSUE THREAD

**Issue by mHvNG** [grahammendick/navigation#738]
The title explains itself mostly. When I add an `barTintColor` I get the shadow beneath the `NavigationBar`. However there's no way to remove it as far as I can see. 

Am I missing something? Or is there some kind of fix inside the AppDelegate file I can use?

**Comment by grahammendick** [comment]
I recommend you use a Material 3 style because the shadow is a Material 2 thing, for example, 
```xml
<style name="AppTheme" parent="Theme.Material3.DayNight.NoActionBar">
```
Material 3 theme will modernise your other components too.  But if you want to stay on Material 2 then a `barTintColor` with an alpha less than 1 removes the shadow, for example, 'rgba(255, 0, 0, 0.99)'

**Comment by mHvNG** [comment]
My apologies, I should've been more clear. I meant iOS. See image. I would like to remove the grey border.
<img width="420" alt="Screenshot 2023-11-02 at 10 47 42" src="https://github.com/grahammendick/navigation/assets/39901096/28906eaf-7c37-4987-9c1b-fe64d34cb5b0">

**Comment by grahammendick** [comment]
Looks like I'm [hiding the shadow if the `barTintColor` is 'transparent'](https://github.com/grahammendick/navigation/blob/b332c63d778e72b909da105eeebe3698aaa02ddf/NavigationReactNative/src/ios/NVNavigationBarView.m#L95).  To be [consistent with android](https://github.com/grahammendick/navigation/blob/b332c63d778e72b909da105eeebe3698aaa02ddf/NavigationReactNative/src/android/src/main/java/com/navigation/reactnative/NavigationBarManager.java#L35) I should also hide it if the `barTintColor` isn't full opaque.

If you'd like to work on a PR let me know. No pressure, of course, just if you're interested

**Comment by grahammendick** [comment]
Also looks like I'm only hiding the shadow for iOS < 13.
For iOS >= 13 I think setting the [shadowColor of UINavigationBarAppearance](https://developer.apple.com/documentation/uikit/uibarappearance/3198008-shadowcolor?language=objc) to nil should work

**Comment by mHvNG** [comment]
@grahammendick Hi, busy afternoon, so didn't have the time to answer. I would introduce a `shadowColor` prop to the component. There are use cases where some screens have the border, but others do not. Flexibility is always nice.

**Comment by mHvNG** [comment]
@grahammendick I'll take a look at it this weekend.

**Comment by grahammendick** [comment]
If you do pick it up at the weekend, I recommend you don't work on the new architecture to start with. Developing is quite a bit tougher in Fabric. It's less painful to iterate/reach agreement on the feature in the old architecture. Let me know if you have any questions.

**Comment by grahammendick** [comment]
@mHvNG I just released navigation-react-native v9.9.0 and [thanked you in the release notes](https://github.com/grahammendick/navigation/releases/tag/v9.9.0-NavigationReactNative)
